src/main/kotlin/no/skatteetaten/aurora/cantus/ApplicationConfig.kt

```diff
@@ -1,24 +1,11 @@
 package no.skatteetaten.aurora.cantus
 
-import org.springframework.beans.factory.config.BeanPostProcessor
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
-import org.springframework.security.crypto.password.PasswordEncoder
-import org.springframework.security.web.authentication.www.BasicAuthenticationEntryPoint
+import org.springframework.web.client.RestTemplate
 
 @Configuration
-class ApplicationConfig : BeanPostProcessor {
-
-    @Bean
-    fun passwordEncoder(): PasswordEncoder {
-        return BCryptPasswordEncoder()
-    }
-
+class ApplicationConfig {
     @Bean
-    fun basic(): BasicAuthenticationEntryPoint {
-        return BasicAuthenticationEntryPoint().also {
-            it.realmName = "CANTUS"
-        }
-    }
+    fun restTemplate() = RestTemplate()
```

Comment:
Man bør egentlig bruke RestTemplateBuilder for å lage RestTemplate, ikke instansiere direkte:

@Bean
fun restTemplate(restTemplateBuilder RestTemplateBuilder) = restTemplateBuilder.build()

Reply:
Virker bedre. Fikset